modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/FR.json

```diff
@@ -320,12 +324,12 @@
 		"regionCode": "53"
 	},
 	{
-		"name": "Mayotte (dept)",
+		"name": "Mayotte",
 		"regionCode": "976"
 	},
 	{
-		"name": "Mayotte (region)",
-		"regionCode": "YT"
```

Comment:
What about the already existing addresses that are using this regioneCode?

Reply:
I also follow the ISO standard to update the region JSON